README.md

```diff
@@ -322,6 +322,7 @@ This repository also provides one such platforms where contributers come over an
 | [Find_the_ball](https://github.com/kunjgit/GameZone/tree/main/Games/Find_the_ball)                  |
 |[Color The Page](https://github.com/kunjgit/GameZone/tree/main/Games/Color_The_Page)|
 |[AquaSort_Game](https://github.com/kunjgit/GameZone/tree/main/Games/AquaSort_Game)   |
+| [Snake](https://github.com/1911aditi/GameZone/tree/cf71e07a9bd67d55607e9ffcd4b9b156fe05d031/Games/snake)                               |
```

Comment:
change the path as other game path shows and only change the end with your game name

Reply:
can you check now.